dvc/config.py

```diff
@@ -162,7 +162,7 @@ class RelPath(str):
                     "endpointurl": str,
                     "access_key_id": str,
                     "secret_access_key": str,
-                    Optional("listobjects", default=False): Bool,
+                    Optional("listobjects", default=False): Bool,  # obsoleted
```

Comment:
Obsoleted since we are using a high-level resource API now, which just does the switch automatically.